### PR TITLE
Added XLMR warmup

### PR DESCRIPTION
**Pull request by erichan1**
stacked on top of #55 

1. Add configurable warmup batches to XLMR. 
2. Added simple readme
3. Removed xlmr_extra.py

### CODE REVIEW

**Review by aaronenyeshi — APPROVED** [review]
LGTM, although this warmup looks like a duplicate of non-warmup. Is there a way we can de-duplicate this?

**Comment by erichan1** [comment]
> LGTM, although this warmup looks like a duplicate of non-warmup. Is there a way we can de-duplicate this?

Good call. I just wanted to get this out quickly originally but I think it's worth it to dedupe.

File: benchmarks/xlmr/ootb/xlmr.py [hunk 1]
```diff
@@ -82,44 +132,37 @@ def run():
...
-    if args.half_model:
+    if device and args.half_model:
         xlmr.half()
     
     # use gpu
-    if args.use_gpu:
+    if device:
         xlmr = xlmr.to(device)
-
+
```
Comment (aaronenyeshi) [inline benchmarks/xlmr/ootb/xlmr.py:141]: please fix this whitespace

**Review by aaronenyeshi — COMMENTED** [review]
LGTM

**Pull request merged**